src/components/Tipseen/TipseenImage.tsx

```diff
@@ -0,0 +1,17 @@
+import cx from "classnames";
+import { VibeComponentProps } from "../../types";
+import { FC } from "react";
+import styles from "./TipseenImage.module.scss";
+
+const BASE_CSS_CLASS = "monday-style-tipseen-image";
+
+interface TipseenImageProps extends VibeComponentProps {
+  src?: string | undefined;
```

Comment:
Both are required, an image without src or alt is not an image or not accessible

Reply:
I know it was not the case before but it will be more accurate :)